pkgs/development/go-modules/generic/default.nix

```diff
@@ -151,6 +151,11 @@ go.stdenv.mkDerivation (
       fi
     }
 
+    export GO_X=""
+    if (( "''${NIX_DEBUG:-0}" >= 1 )); then
+      export GO_X="-x"
```

Comment:
Please don't export the variable.

Reply:
Use `buildFlagsArray+=(-x)` instead of the variable.

Reply:
Done